lib/run/summary.js

```diff
@@ -38,6 +38,16 @@ RunSummary = function RunSummary (emitter, options) {
          */
         globals: _.get(options, 'globals'),
 
+        /**
+         * Holds information related to the reporters specified.
+         */
+        reporters: _.get(options, 'reporters', []),
```

Comment:
Should/would this always be an array? Even if one reporter is specified? Can we add a test for this as well?